instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -18,19 +72,21 @@ class SearchBar extends Component {
 
     render() {
         return(
-            <div className="header">
-                <div className="logo">
+            <HeaderDiv>
+                <LogoDiv>
                     <i className="fab fa-instagram"></i>
-                    <span>|</span>
-                    <img src="instagram_logo.svg" />
-                </div>
+                    <Line />
+                    <img className="logo-img" src="instagram_logo.svg" alt="logo" />
+                </LogoDiv>
                 
-                <input className="searchbar" type="text" placeholder="Search" value={this.state.searchQuery} onChange={this.changeHandler}></input>
+                <SearchbarInput type="text" placeholder="Search" value={this.state.searchQuery} onChange={this.changeHandler}></SearchbarInput>
```

Comment:
This should be broken up for readability (and especially for avoiding horizontal scroll):
```javascript
<SearchbarInput 
  type="text" 
  placeholder="Search" 
  value={this.state.searchQuery} 
  onChange={this.changeHandler}
>
</SearchbarInput>
```